recipes/pydtmc/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "PyDTMC" %}
+{% set version = "6.2.0" %}
+{% set user = "TommasoBelluzzo" %}
+
+package:
+  name: {{ name.lower() }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4ea910105452b495721ebebd64163e5a33e42d82259e9b8d4c1c1fb6cfd4c293
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    - setuptools
+  host:
+    - python >=3.6
+    - pip
```

No need for the `build` section:
```suggestion
  host:
    - python >=3.6
    - pip
    - setuptools
```